packages/web/src/components/ui/PackagePreview/PackagePreview.tsx

```diff
@@ -182,75 +277,47 @@ export default function PackagePreview({ name, version, opened }: Props) {
               </div>
 
               <div className={styles.popularity}>
-                <div className={styles.popularityItemWrapper}>
-                  <div className={styles.popularityItem}>
-                    <div className={styles.popularityFill} style={{ height: '100%' }}>
-                      89 912
+                {modules.map(({ fill, uses, moduleVersion, bug }) => (
+                  <div className={styles.popularityItemWrapper}>
+                    <div className={styles.popularityItem}>
+                      {packageDetailsLoading ? (
+                        <div
+                          className={clsx(styles.popularityFill, styles.popularityFillSkeleton)}
+                          style={{ height: fill }}
```

Comment:
We should be on lookout for these deoptimizations, this should definitely be a component / top-level const.

Reply:
Added as `TODO` for future `PackagePreview` refactor.